Documentation/Books/Manual/Programs/Arangodump/Maskings.md

```diff
@@ -61,8 +65,78 @@ For example:
     }
 
 In the example the collection "private" is completely ignored. Only the
-structure of the collection "log" is dump, but not the data itself.
-The collection "person" is dumped completed but masking the "name" field when
-it occurs on the to-level. It masks the field "security_id" anywhere in the
-document.
+structure of the collection "log" is dumped, but not the data itself.
+The collection "person" is dumped completely but masking the "name" field if
+it occurs on the top-level. It masks the field "security_id" anywhere in the
+document. See below for a complete description of the parameters of
+"xify_front".
+
+Path
+----
+
+If the path starts with a `.` then it is considered to be a wildcard match.
+For example, `.name` will match the attribute name `name` everywhere in the
+document. `name` will only match at top level. `person.name` will match
+the attribute `name` in the top-level object `person`.
+
+If you have a attribute name that contains a dot, you need to quote the
+name with either a tick or a backtick. For example
+
+    "path": "´name.with.dots´"
+
+or
+
+    "path": "`name.with.dots`"
+
+xify_front
+----------
+
+This masking replaces characters with `x` and ` `. Alphanumeric characters,
```

Comment:
Leading/trailing whitespace in inline code blocks is stripped on Github, but not in Gitbook in my test, so this should be fine.